hbp_knowledge/nfkb/srinivasan2015.bel

```diff
@@ -74,133 +74,175 @@ SET Citation = {"PubMed", "25652642"}
+SET Confidence = "Medium"
 SET MeSHAnatomy = {"Cerebral Cortex","Hippocampus"}
-complex(p(HGNC:NFKB1),p(HGNC:RELA),a(GO:"NF-kappaB complex")) -> act(complex(GO:"NF-kappaB p50/p65 complex"))
+complex(p(HGNC:NFKB1),p(HGNC:RELA),a(GO:"NF-kappaB complex"), loc(GO:nucleus)) -> act(complex(GO:"NF-kappaB p50/p65 complex"))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "In conditional neuronal NF-κB-deficient mice, loss of NF-κB signaling impaired synaptic
 transmission, spatial memory formation, and plasticity"
+SET Confidence = "Medium"
+SET Species = "10090"
 act(complex(GO:"NF-kappaB complex")) -> bp(MESH:"Synaptic Transmission")
 act(complex(GO:"NF-kappaB complex")) -> path(MESH:"Spatial Memory")
 act(complex(GO:"NF-kappaB complex")) -> bp(GO:"positive regulation of synaptic plasticity")
+UNSET Species
+UNSET Confidence
 
 SET Evidence = "A number of physiological stimuli including membrane depolarization or glutamergic signal
 transduction lead to rapid activation of the inducible NF-κB localized in the synapses,
 cytoplasm and dendrites of the neurons"
+SET Confidence = "High"
 SET MeSHAnatomy = {"Synapses","Cytoplasm","Dendrites","Neurons"}
 bp(GO:"membrane depolarization") -> act(complex(GO:"NF-kappaB complex"))
 bp(GO:"synaptic transmission, glutamatergic") -> act(complex(GO:"NF-kappaB complex"))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Complete abrogation of the DNA binding ability of NFκB factors induces apoptosis of the neuronal cells"
+SET Confidence = "Medium"
 SET MeSHAnatomy = "Neurons"
-act(complex(GO:"NF-kappaB complex")) -| bp(GO:"apoptotic process")
+act(complex(GO:"NF-kappaB complex"), ma(GO:"DNA binding")) -| bp(GO:"apoptotic process")
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = " Cell death is preceded by reduction in the
 NF-κB regulated transcription of anti-apoptotic genes suggesting that a minimal threshold of
 NF-κB activity is needed for neuronal survival"
+SET Confidence = "High"
 act(complex(GO:"NF-kappaB complex")) -| bp(GO:"neuron death")
+UNSET Confidence
 
 SET Evidence = "Several kinase pathways
 including the calcium-calmodium dependent kinase-II (CaMK), the protein kinases-C (PKC)
 and the ras/phosphatidylinositol 3-kinase (PI3K) pathways have been implicated in
 activating neuronal NF-κB signaling"
+SET Confidence = "High"
 SET MeSHAnatomy = "Neurons"
 bp(GO:"calmodulin dependent kinase signaling pathway") -> act(complex(GO:"NF-kappaB complex"))
 act(p(FPLX:PKC)) -> act(complex(GO:"NF-kappaB complex"))
 bp(GO:"phosphatidylinositol 3-kinase signaling") -> act(complex(GO:"NF-kappaB complex"))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Unlike neurons, NF-κB is present in the cytoplasm as an
 inactive complex with the IκB proteins in glial cells under physiological conditions"
+SET Confidence = "Low"
 SET MeSHAnatomy = {"Cytoplasm","Neuroglia"}
 a(MESH:"I-kappa B Proteins") -| act(complex(GO:"NF-kappaB complex"))
```

Comment:
this seems like it shouldn't be listed as an abundance. Does this pass semantic validation? what's the mesh ID for this? Maybe we can add more rules to the bio2bel MeSH output

Reply:
According to my understanding of the statement, it seems as if IkB and NF-kB form a complex which makes MF-kB inactive.

Reply:
First, never use mesh for protein families. We have famplex for that. Switch to `p(FPLX:IKB)`

fine. so `complex(complex(GO:"NF-kappaB complex"), p(FPLX:IKB)) =| act(complex(GO:"NF-kappaB complex"))`